### PR TITLE
Integrate the gopath hacks

### LINKED ISSUE #250: Use GOPATH hacks to codegen anywhere.
Please see https://github.com/knative/eventing/pull/4621 or https://github.com/knative/serving/pull/10245 for an example.

### PR DESCRIPTION
**Pull request by arghya88**
Signed-off-by: Arghya Sadhu <arghya88@gmail.com>

Fixes #250

## Proposed Changes

- Use GOPATH hacks to codegen anywhere.

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -18,22 +18,18 @@ set -o errexit
...
-if [ -z "${GOPATH:-}" ]; then
-  export GOPATH=$(go env GOPATH)
-fi
-
-source $(dirname $0)/../vendor/knative.dev/hack/library.sh
+echo "=== Update Codegen for $MODULE_NAME"
 
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${REPO_ROOT_DIR}; ls -d -1 $(dirname $0)/../vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
 
 KNATIVE_CODEGEN_PKG=${KNATIVE_CODEGEN_PKG:-$(cd ${REPO_ROOT_DIR}; ls -d -1 $(dirname $0)/../vendor/knative.dev/pkg 2>/dev/null || echo ../pkg)}
```
Comment (n3wscott) [inline hack/update-codegen.sh:30]: you can drop these two lines.
/hold

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**